pennylane_forest/device.py

```diff
@@ -188,35 +188,21 @@ class ForestDevice(QubitDevice):
     def __init__(self, wires, shots=1000, analytic=False, **kwargs):
         super().__init__(wires, shots, analytic=analytic)
         self.analytic = analytic
-        self.forest_url = kwargs.get("forest_url", pyquil_config.forest_url)
-        self.qvm_url = kwargs.get("qvm_url", pyquil_config.qvm_url)
-        self.compiler_url = kwargs.get("compiler_url", pyquil_config.quilc_url)
-
-        self.connection = ForestConnection(
-            sync_endpoint=self.qvm_url,
-            compiler_endpoint=self.compiler_url,
-            forest_cloud_endpoint=self.forest_url,
-        )
```

Comment:
Creating a connection is now needed _before_ `init` is called, so we make it a static method.

Reply:
🤔